src/app/settings/settings.effects.spec.ts

```diff
@@ -0,0 +1,89 @@
+import { TestBed } from '@angular/core/testing';
+import { provideMockActions } from '@ngrx/effects/testing';
+import { Action } from '@ngrx/store';
+import { EffectsMetadata, getEffectsMetadata } from '@ngrx/effects';
+import { Observable } from 'rxjs';
+import { cold } from 'jasmine-marbles';
+
+import { SettingsEffects } from './settings.effects';
+import { AnimationsService, LocalStorageService } from '@app/core';
+import {
+  SettingsActionTypes,
+  SETTINGS_KEY
+} from '@app/settings/settings.reducer';
+
+describe('SettingsEffects', () => {
+  let actions$: Observable<Action>;
+  let effects: SettingsEffects;
+  let metadata: EffectsMetadata<SettingsEffects>;
+  let localStorageService: any;
+  let animationsService: any;
+
+  beforeEach(() => {
+    TestBed.configureTestingModule({
+      providers: [
+        SettingsEffects,
+        provideMockActions(() => actions$),
+        {
+          provide: LocalStorageService,
+          useValue: jasmine.createSpyObj('LocalStorageService', ['setItem'])
+        },
+        {
+          provide: AnimationsService,
+          useValue: jasmine.createSpyObj('AnimationsService', [
+            'updateRouteAnimationType'
+          ])
+        }
+      ]
+    });
+
+    localStorageService = TestBed.get(LocalStorageService);
+    animationsService = TestBed.get(AnimationsService);
+    effects = TestBed.get(SettingsEffects);
+  });
+
+  it('should not dispatch any action', () => {
```

Comment:
The reason why I would put it inside the block is because it's still testing the decorator of `persistSettings`.
If another effect would be added it would seem weird to me that this test is outside of the `persistSettings` block.

But after reading your explanation I'm also getting your point 😄

Reply:
@timdeschryver  OK. All tests should be in one block now.